src/foam/nanos/auth/PasswordExpiryAuthService.js

```diff
@@ -41,8 +41,10 @@ foam.CLASS({
     },
     {
       name: 'login',
-      javaCode:
-        `User user = (User) ((DAO) getLocalUserDAO())
+      javaCode:`
+        // on login, clear the context spid to avoid spid restrictions on user find
+        x = x.put("spid", null);
```

Comment:
Might be hard to reason later why clearing spid in PasswordExpiryAuthService, maybe create a decorator for just to do that and have it as the first decorator.

Reply:
True, will do that, thanks